tldw_Server_API/tests/Services/test_executor_registry_shutdown.py

```
@@ -0,0 +1,74 @@
+
+
+pytestmark = pytest.mark.unit
+
+
+class _BlockingExecutor(Executor):
+    def __init__(self, started: threading.Event, release: threading.Event, started_count: list[int]) -> None:
+        self._started = started
+        self._release = release
+        self._started_count = started_count
+
+    def shutdown(self, wait: bool = True, cancel_futures: bool = True) -> None:
```

Ruff (0.15.7)

[warning] Boolean-typed positional argument in function definition

(FBT001)